.github/actions/deploy_docs/action.yml

```diff
@@ -0,0 +1,46 @@
+name: 'Deploy documentation'
+description: 'Deploy ESPResSo documentation'
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        # checkout github pages
+        cd "${HOME}"
+        git clone --quiet git@github.com:espressomd/espressomd.github.io.git
+        cd espressomd.github.io
+        LAST_COMMIT=$(git log -1 --pretty=format:"%s" remotes/origin/main -- doc dox tutorials)
```

Comment:
Please add  a comment explaining which commit is or isn't selected, here

Reply:
After further reflection, we don't actually need to filter files here, because the commit we're looking for is always the most recent one. I'll remove the superfluous `-- doc dox tutorials`.